src/OpenTracing/ISpan.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace OpenTracing
+﻿namespace OpenTracing
 {
+    using System;
+    using System.Collections.Generic;
```

Comment:
Why would one place the usings inside of the namespace block? Also isn't this a change to this repo's existing style and formatting....

Reply:
Hi, You can see here a good explanation:
https://stackoverflow.com/questions/125319/should-using-statements-be-inside-or-outside-the-namespace

But you know, you can do that way. It's an opinion ;) Thanks.